napari/components/overlays/bounding_box.py

```diff
@@ -30,7 +30,7 @@ class BoundingBoxOverlay(SceneOverlay):
 
     lines: bool = True
     line_thickness: float = 1
-    line_color: ColorValue = 'red'
+    line_color: ColorValue = ColorValue('red')
```

Comment:
Unfortunately, I think you'll need to use the default factory to satisfy the min-requirement test, as discussed in a previous PR: https://github.com/napari/napari/pull/5871#issuecomment-1613536565